pkg/skaffold/util/util.go

```diff
@@ -213,3 +215,31 @@ func BoolPtr(b bool) *bool {
 	o := b
 	return &o
 }
+
+func ReadConfiguration(filename string) ([]byte, error) {
+	switch {
+	case filename == "":
+		return nil, errors.New("filename not specified")
+	case filename == "-":
+		return ioutil.ReadAll(os.Stdin)
+	case strings.HasPrefix(filename, "http://") || strings.HasPrefix(filename, "https://"):
+		return Download(filename)
+	default:
+		return ioutil.ReadFile(filename)
+	}
+}
+
+func Download(url string) ([]byte, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	return ioutil.ReadAll(resp.Body)
+}
+
+func WriteNewConfiguration(config []byte, originalFilename string) error {
```

Comment:
remove this

Reply:
This is used to write out the newly generated config